### ISSUE TITLE
Duplicated XML IDs in the same file

### ISSUE THREAD

**Issue by andreparames** [odoo/odoo#18513]
Impacted versions: 10.0

The file` addons/crm/report/crm_opportunity_report_views.xml` has the same XML ID on two different records:

```xml
<record id="ir_filters_crm_opportunity_report_next_action" model="ir.filters">
    <field name="name">Expected Revenus by Team</field>
    <field name="model_id">crm.opportunity.report</field>
    <field name="user_id" eval="False"/>
    <field name="context">{'group_by': ['create_date:month', 'team_id'], 'measures': ['expected_revenue', '__count']}</field>
</record>
<record id="ir_filters_crm_opportunity_report_next_action" model="ir.filters">
    <field name="name">Team Activities</field>
    <field name="model_id">crm.opportunity.report</field>
    <field name="user_id" eval="False"/>
    <field name="context">{'group_by': ['team_id'], 'measures': ['nbr_activities']}</field>
</record>
```

https://github.com/odoo/odoo/blob/10.0/addons/crm/report/crm_opportunity_report_views.xml#L80:L91

(Also, spelling of "Revenus" in the first record)

**Comment by odony** [comment]
Thanks, this issue is present in 9.0 too and stems from rev. a8c5b7810f412d2b956f8163e169159471494ea5.

**Comment by Yenthe666** [comment]
@odony if you give me the name of how you'd like to name one of both I'll make a PR for you.

**Comment by odony** [comment]
Fixed by 586b97e17bc595607c4a88ffc84a0f8a02147520, will be forward-ported to 10.0 soon

@Yenthe666 thanks, I had the local diff but hadn't pushed it yet ;-)

**Comment by Yenthe666** [comment]
Good job - thanks!
